ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/autovpn-rr1.cfg

```diff
@@ -42,10 +41,10 @@ router path-selection
       path-group INET
    !
    policy DEFAULT-AVT-POLICY-WITH-CP
+      default-match
```

Comment:
The order on EOS is opposite I believe? default-match being last.

Reply:
not for router path-selction